### DISCUSSION TITLE
backspace behaves weirdly in vi mode

### DISCUSSION THREAD

**Discussion by pdeva** [ghostty-org/ghostty#3523] [Q&A] [answered]
when enabled via `bindkey -v` the backspace key either doesnt delete at all or deletes only recently typed chars.

video showing the issue:
https://www.loom.com/share/9ac1f623f52049038ce7b883c163e63f

in the video, i am constantly pressing backspace on my keyboard but its not doing anything. 

it seems that when i go into insert mode and type new chars, the backspace works only on those specific chars. if i then leave insert mode and then enable it again, the chars i typed previously are no longer able to be deleted by backspace.

**Comment by mitchellh** [comment]
Pretty confident this is a dupe but triaging right now.

**Reply by Integralist** [reply, marked as answer]
This looks/sounds like what I reported here https://github.com/ghostty-org/ghostty/discussions/3487

**Reply by mitchellh** [reply]
That's it. Thanks
